sklearn/neighbors/lof.py

```diff
@@ -85,7 +85,7 @@ class LocalOutlierFactor(NeighborsBase, KNeighborsMixin, UnsupervisedMixin):
 
     p : integer, optional (default=2)
         Parameter for the Minkowski metric from
-        :ref:`sklearn.metrics.pairwise.pairwise_distances`. When p = 1, this is
```

Comment:
I think this should work if you change `:ref:` to `:func:`, and I would much prefer that.